.yamato/config/universal_2d.metafile

```diff
@@ -29,7 +26,9 @@ test_platforms:
           - [OSX_Metal]: --timeout=2400
       - [Android_OpenGLES3, Android_Vulkan]: <shared.platform.commands.kill_server>
     commands_build:
-      - <reuse_blocks.setup_without_utr>
+      - [Win_DX11, Win_DX12, Win_Vulkan]: cd Tools && powershell -command ". .\Unity.ps1; Set-ScreenResolution -width 1920 -Height 1080"
```

Comment:
I _think_ we don't technically need the screen resolution command in build, but it also shouldn't affect the job, so I'm fine keeping it for now.

Reply:
Great spot, thanks @kivistikliis🥇 Looks like they were 'unrolled' in https://github.com/Unity-Technologies/Graphics/commit/3842e4c6cdecca20f75e21fa61cc8667f51d5e6a (to keep the the generated yaml the same as before), I removed them (from `commands_build`) in https://github.com/Unity-Technologies/Graphics/pull/7218/commits/dd0b56b1ce676cc0ba7e7338d57a94abd953b7bb and will kick off another testing pipeline!

Reply:
Awesome, thanks!